recipes/folly/all/patches/0014-find-librt.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/CMake/FollyConfigChecks.cmake b/CMake/FollyConfigChecks.cmake
+index 6b8b308c7..908d72d51 100644
+--- a/CMake/FollyConfigChecks.cmake
++++ b/CMake/FollyConfigChecks.cmake
+@@ -83,6 +83,13 @@ string(REGEX REPLACE
+   CMAKE_REQUIRED_FLAGS
+   "${CMAKE_REQUIRED_FLAGS}")
+ 
++if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
++  find_library(LIBRT rt)
++  if (LIBRT)
++    set(CMAKE_REQUIRED_LIBRARIES "rt")
```

Comment:
Should `rt` be appended to the list, or replace it?
```suggestion
+    list(APPEND CMAKE_REQUIRED_LIBRARIES "rt")
```

Reply:
`CMAKE_REQUIRED_LIBRARIES` is used for other libraries like zlib and OpenSSL: https://github.com/facebook/folly/search?q=CMAKE_REQUIRED_LIBRARIES.  I didn't want to disturb those.

Reply:
By doing a plain `set`, you overwrite the previous content.

Reply:
Doh.  That's what I had intended to to.  Fixed.

Reply:
Should `rt` also be added `system_libs` in folly's conanfile.

Reply:
Yes - done.